Logging/Log.cs

```diff
@@ -140,6 +140,25 @@ public static void Error(string format, params object[] args)
         }
 
         /// <summary>
+        /// Log trace with format string
+        /// </summary>
+        public static void TraceFormat(string traceFormat, params object[] args)
```

Comment:
I like this, but I wonder if we can just have an overload of `Log.Trace`:
```C#
public static void Trace(string message) { .. }
public static void Trace(string format, params object[] args) { ... }
```

I know R# will say overload hidden by params, but in this case it would be OK?

Thoughts?